lib/restrictions.js

```diff
@@ -5,6 +5,7 @@ import * as bedrock from 'bedrock';
 import * as database from 'bedrock-mongodb';
 import {promisify} from 'util';
 import {ResourceRestriction} from './ResourceRestriction.js';
+import assert from 'assert-plus';
```

Comment:
Please move this up to the top (alphabetize).

Reply:
Fixed.